modules/data/src/main/java/io/fluo/webindex/data/FluoApp.java

```diff
@@ -28,8 +28,6 @@
 
 public class FluoApp {
 
-  public static final int NUM_BUCKETS = 119;
```

Comment:
I think you said something before that this number cannot be changed for an existing application.  Is that still true?  If so, a comment should be added to `data.yml.example` about this.